packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -4,9 +4,10 @@ import { FileUploaderQueueLimitBehaviorType, Validation } from '../../types';
 
 export interface FileUploaderStyleProps extends Validation {
   isDragAndDropSupported?: boolean;
-  // isDisabled?: boolean;
+  isDisabled?: boolean;
   isDragging?: boolean;
   isDropZoneHidden?: boolean;
+  isInputDisabled?: boolean;
```

Comment:
Why two props for the same thing? 🤔

Reply:
Because it could be either triggered by the isDisabled prop from the root or by filling the queue. I wasn't able to do it any other way, but maybe I missed something.